### PR TITLE
DOC-93 New project types page with addition of graphics

### PR DESCRIPTION
**Pull request by rmalik-es**
Added new designed graphics and new content based on project types and previous feedback

Continuation of https://github.com/EventStore/documentation/pull/627

Thanks

### CODE REVIEW

File: docs/project-types.md [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+
+
+## 1. Brownfield Legacy Migration
+Brownfield legacy migration projects involve the process of upgrading, modernizing, or completely overhauling existing legacy systems with event-sourced architectures.
+
+
+![Brownfield legacy migration graphic](docs/images/brownfield-project-type.png)
+
+
+## **Characteristics**
```
Comment (rsowald) [inline docs/project-types.md]: This should probably be an h3

File: docs/project-types.md [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+- **Strangulation pattern:** A common strategy in brownfield migration is the strangler pattern, where new functionality gradually replaces old functionality. This is a gradual transfer of functionality from old systems to the new event-sourced model.
+
+
+## 2. Greenfield projects
+Greenfield projects involve projects that start from scratch, without any existing constraints from prior work. It’s ideal for implementing modern, event-driven architectures from the ground up. 
+
+![Greenfield project type graphic](docs/images/greenfield-project-type.png)
+
+
+## **Characteristics**
```
Comment (rsowald) [inline docs/project-types.md]: again, should be an h3

File: docs/project-types.md [hunk 3]
```diff
@@ -0,0 +1,69 @@
...
+
+
+
+## 3. Brownfield Sidecar Source of Truth
+Sidecar approaches refer to a strategy of integrating a new system (in this case, an event store) alongside existing legacy systems. This approach is designed to enhance or augment the existing infrastructure without disruption. 
+
+![Brownfield sidecar project type graphic](docs/images/brownfield-sidecar-project-type.png)
+
+
+## **Characteristics**
```
Comment (rsowald) [inline docs/project-types.md]: h3

**Comment by rsowald** [comment]
@rmalik-es The graphics look great, AFAIK, but we may need to move them and reference them a little differently. With the way Vite (the VuePress site generator) works, the images in these docs in the top level docs directory may not be loaded before the md file. So, we could make the images folder an external module in the config, but the other way (and the way our getting started images work) is to move all the images into the `/public` directory under `/.vuepress`. You can leave them in their parent folder if you want, or just put the images in that public folder and delete your empty folder. Then you can reference the image like `Brownfield legacy migration graphic`.

**Comment by rmalik-es** [comment]
> @rmalik-es The graphics look great, AFAIK, but we may need to move them and reference them a little differently. With the way Vite (the VuePress site generator) works, the images in these docs in the top level docs directory may not be loaded before the md file. So, we could make the images folder an external module in the config, but the other way (and the way our getting started images work) is to move all the images into the `/public` directory under `/.vuepress`. You can leave them in their parent folder if you want, or just put the images in that public folder and delete your empty folder. Then you can reference the image like `Brownfield legacy migration graphic`.

Great idea, thanks!

**Review by rsowald — APPROVED** [review]

**Pull request merged**